xtask/src/cli.rs

```diff
@@ -22,6 +22,13 @@ pub mod test {
         #[clap(long, default_value = "false")]
         pub ci: bool,
 
+        /// If the test is executed locally
+        ///
+        /// This flag will not require to add ikura-node, ikura-shim, and sov-demo-rollup
+        /// to the path. This will work ONLY if you keep the default CARGO_TARGET_DIR
+        #[clap(long, default_value = "false")]
+        pub local: bool,
```

Comment:
you can use `conflicts_with = "ci"` to enforce that they are mutually exclusive at parse-time, FYI

Reply:
Oh, thanks! I didn't know this! I will use it for sure!